src/components/ReportActionItem/MoneyRequestView.js

```diff
@@ -349,5 +403,15 @@ export default compose(
                 return `${ONYXKEYS.COLLECTION.TRANSACTION}${transactionID}`;
             },
         },
+        transactionViolation: {
```

Comment:
there's a typo here which I'm fixing [here](https://github.com/Expensify/App/pull/34479/files) if you can review please @trevor-coleman . I was able to see the violations once I fixed it.